File: files/CH-049.py
import pandas as pd

input = pd.read_excel("CH-049 Assignment Problem Part 1.xlsx", usecols="C:F", skiprows=1).values
test = pd.read_excel("CH-049 Assignment Problem Part 1.xlsx", usecols="P:S", skiprows=1).values

input = [[val - min(row) for val in row] for row in input]
input = list(map(list, zip(*input)))
input = [[val - min(row) for val in row] for row in input]
input = list(map(list, zip(*input)))

print(input == test)    # True

File: files/CH-049.R
library(tidyverse)
library(readxl)

input = read_excel("files/CH-049 Assignment Problem Part 1.xlsx", range = "C2:F6") 
step2 = read_excel("files/CH-049 Assignment Problem Part 1.xlsx", range = "P2:S6")

result = input %>%
  rowwise() %>%
  mutate(across(everything(), ~ . - min(c_across(everything())))) %>%
  ungroup() %>%
  mutate(across(everything(), ~ . - min(.)))

identical(result, step2)
#  [1] TRUE